tests/suite/test_agent_app_protect.py

```diff
@@ -18,7 +18,7 @@
     ],
     indirect=["crd_ingress_controller_with_ap"],
 )
-class TestAppProtectAgent:
+class TestAppProtectAgentV2:
```

Comment:
```suggestion
@pytest.mark.agentv2
```
and then same change in https://github.com/nginx/kubernetes-ingress/blob/main/.github/data/matrix-smoke-nap.json#L72

Reply:
updated.